tests/unit/AboutTheBusiness.spec.ts

```diff
@@ -0,0 +1,57 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import VueRouter from 'vue-router'
+import { shallowMount } from '@vue/test-utils'
```

Comment:
Whenever possible, in unit tests, I think it's better to deep-mount the component under test.

Pro: Vuetify sub-components render and function correctly (they are not stubbed).

Con: All other sub-components are rendered and this may complicate the testing. Remember, we want to test THIS component, not everything under it (which should have their own tests). If you need a mix of Vuetify but not other sub-components, you can stub them out like this:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/e2a94ef7-4f77-4a00-b11e-d86975763c34)